pom.xml

```diff
@@ -43,13 +47,11 @@
 
     <properties>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
-        <confluent.common.version>3.2.1-SNAPSHOT</confluent.common.version>
-        <slf4j.version>1.7.21</slf4j.version>
+        <confluent.common.version>${project.version}</confluent.common.version>
```

Comment:
i think its fine to leave like this now. say at some point we move off of snapshot to independently versioned stuff